after/ftplugin/xml.vim

```diff
@@ -1,12 +1,6 @@
 if gradle#isGradleProject()
-
-  call gradle#setClassPath()
-  call gradle#setupGradleCommands()
-
   if android#isAndroidProject()
-    call android#setAndroidSdkTags()
-    call android#setClassPath()
-    call android#setupAndroidCommands()
+    autocmd BufEnter *.xml XMLns android
+    autocmd BufEnter AndroidManifest.xml XMLns manifest
```

Comment:
All that I did to generate these was download the android sdk sources and generate a schema file based on all the samples, then convert the schema file to a vim xml completion file. Unfortunately, this means the XML completion we have here is ultimately pretty stupid and I didn't want to confuse it

There's a theoretical concern with the completions getting mixed up if these files were merged, but I haven't tinkered enough to be absolutely certain